synthetic_data/marginal_dist.py

```diff
@@ -70,13 +70,16 @@ def _detect_dist_discrete(col_stats):
     observed_freq = [categorical_count[category] for category in categories]
     sample_size = col_stats["sample_size"]
 
+    # Shift categories so they start at 0
+    categories = list(range(len(categories)))
```

Comment:
why is this needed? Overwriting this variable with the same variable name is a bit of an alarm for me